### DISCUSSION TITLE
New build error: importTransformation is not a function

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#17529] [General - Components / Directives / etc] [answered]
Just upgraded to quasar v2.17.0 and now getting this error:
Syntax Error: TypeError: importTransformation is not a function
    at ...node_modules\@quasar\app-webpack\lib\webpack\loader.vue.auto-import-quasar.js:20:45

**Comment by rstoenescu** [comment, marked as answer]
@CrawfordW Please upgrade the CLI as well. It's been documented on the release notes as well for Quasar v2.16 + CLIs
